### PR TITLE
Fix github url strings (org edx -> openedx)

### PR DESCRIPTION
**Pull request by sarina**
## This PR was autogenerated

This pr replaces the old GitHub organization, github.com/edx, with the new GitHub organization, github.com/openedx.

Tagging @openedx/tcril-engineering for review, but others are welcome to provide review.

Ref: https://github.com/openedx/tcril-engineering/issues/42

### CODE REVIEW

File: cookiecutter-django-app/{{cookiecutter.repo_name}}/pylintrc [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # ** DO NOT EDIT THIS FILE **
```
Comment (timmc-edx) [inline cookiecutter-django-app/{{cookiecutter.repo_name}}/pylintrc:2]: There are a couple more pylintrc files down inside the templates.

File: docs/decisions/0001-combined-cookiecutter-repo.rst [hunk 2]
```diff
@@ -31,6 +31,6 @@ References
 
 Archived cookiecutters:
 
-* https://github.com/edx/cookiecutter-django-app
-* https://github.com/edx/cookiecutter-django-ida
-* https://github.com/edx/cookiecutter-xblock
+* https://github.com/openedx/cookiecutter-django-app
+* https://github.com/openedx/cookiecutter-django-ida
+* https://github.com/openedx/cookiecutter-xblock
```
Comment (timmc-edx) [inline docs/decisions/0001-combined-cookiecutter-repo.rst:36]: These are actually in edx-unsupported at the moment.

Comment (timmc-edx) [reply docs/decisions/0001-combined-cookiecutter-repo.rst:36]: (Same for ADR 3)

**Comment by sarina** [comment]
Thanks for the expert knowledge, @timmc-edx . I found these files:
```
./cookiecutter-django-app/{{cookiecutter.repo_name}}/pylintrc
./python-template/{{cookiecutter.placeholder_repo_name}}/pylintrc
./pylintrc
./cookiecutter-django-ida/{{cookiecutter.repo_name}}/pylintrc
```
and have run `edx_lint write pylintrc` on them.

File: python-template/{{cookiecutter.placeholder_repo_name}}/.editorconfig [hunk 3]
```diff
@@ -2,7 +2,7 @@
 # ** DO NOT EDIT THIS FILE **
```
Comment (timmc-edx) [inline python-template/{{cookiecutter.placeholder_repo_name}}/.editorconfig:2]: Another one! :dizzy_face:

Comment (sarina) [reply python-template/{{cookiecutter.placeholder_repo_name}}/.editorconfig:2]: oh noooo the curse of `.editorconfig` 😆

Comment (sarina) [reply python-template/{{cookiecutter.placeholder_repo_name}}/.editorconfig:2]: thankfully just the one! easy and fast, pushed change.

**Review by timmc-edx — APPROVED** [review]

**Comment by timmc-edx** [comment]
All looks good now, thank you!

**Pull request merged**